tools/update_default_release_versions_to_latest.py

```diff
@@ -503,6 +500,19 @@ def main(args):
 
         unhold_pr(github_pr)
```

Comment:
perhaps this should happen at the bottom, once all of the other steps succeed.

(GH comments are dumb)